packages/app/lib/common/index.js

```diff
@@ -214,8 +214,8 @@ const mapOfDeprecationReplacements = {
 };
 
 const v8deprecationMessage =
-  'This v8 method is deprecated and will be removed in the next major release ' +
-  'as part of move to match Firebase Web modular v9 SDK API.';
+  'This method is deprecated (as well as all React Native Firebase namespaced API) and will be removed in the next major release ' +
+  'as part of move to match Firebase Web modular v9 SDK API. Please see migration guide for more details: https://rnfirebase.io/migrating-to-v22';
```

Comment:
This was called out as confusing - using "v9" at all - js-sdk is on v11 now anyway, and was it v9 of react-native-firebase, but we're going to v21? With enough context it makes sense but I think just strictly using "namespaced" and "non-modular" is good enough

```suggestion
  'as part of move to match Firebase Web modular SDK API. Please see migration guide for more details: https://rnfirebase.io/migrating-to-v22';
```